packages/model-runner/src/main.ts

```diff
@@ -53,13 +53,14 @@ async function main() {
     const inputData = fs.readFileSync(inputFilename, 'utf8')
     const input = JSON.parse(inputData) as RequestInput
 
-    enforceInputSchema(input)
+    enforceRunnerInputSchema(input)
 
     inputID = input.id
     callbackURL = input.callbackURL
     logger.info(input.configuration)
-    modelSlug = input.configuration.model.slug
-    const dockerImage = input.configuration.model.imageURL
+    const model = input.models.shift()
```

Comment:
Control plane update needed to make this work, since it will currently set `input.configuration.model` but leave `input.models` unchanged.
Do you want to enforce here that `input.models` has length 1?

Reply:
I am on the fence about how strongly to enforce that here. So I guess we should probably err on the side of caution.